frontend/index.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
```

Comment:
So we have added no css at all it seems, right?

Reply:
eg it's all pointing to a third party server which holds the css or somehow the css is within the package installed in the container?

Reply:
<img width="424" alt="image" src="https://github.com/openpredictionmarkets/socialpredict/assets/13304149/f12c31f2-f86b-4b2d-8e1b-220499f8724e">